vendor/github.com/Shopify/sarama/.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 go:
 - 1.4.3
-- 1.5.2
```

Comment:
Why did `Shopify/sarama` files change? I don't see any corresponding changes in `Godeps.json`